package.json

```diff
@@ -10,7 +10,9 @@
   },
   "scripts-info": {
     "jsonschema2md": "Display the help for jsonschema2md",
-    "render_schema_docs": "Render the JSON schemas in `schemas` folder to documentation in `docs/schemas`."
+    "render_schema_docs": "Render the JSON schemas in `schemas` folder to documentation in `docs/schemas`.",
+    "validate_markdown_links": "Checks if all links refered to in the documentation are still working. Tends to get rate limited by github",
+    "lint_markdown": "Warnings about markdown styling. Should not be taken as an absolute source of truth"
```

Comment:
We should probably add `"info": "npm-scripts-info"` under `scripts` as described [here](https://www.npmjs.com/package/npm-scripts-info) so that the descriptions under `scripts-info` can be displayed. But it's probably best if I open another PR for that as it's out of the scope of this one.

Reply:
I filed #870 to track this